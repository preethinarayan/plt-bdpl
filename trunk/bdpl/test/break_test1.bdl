int i;
i=0;
if(i<0){
	break;
}
else{
	continue;
}