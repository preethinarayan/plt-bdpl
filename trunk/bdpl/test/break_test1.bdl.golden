'continue' is only permitted within the body of a loop
